File: sdk/php/src/Model/ClickCaptchaResponse.php
<?php

namespace HJTPX\Captcha\Model;

class ClickCaptchaResponse
{
    public $sessionId;
    public $imageUrl;
    public $hint;
    public $hintOrder;
    public $maxPoints;
    public $mode;
    public $allowShuffle;
    public $points;

    public function __construct(array $data)
    {
        $this->sessionId = $data['session_id'] ?? null;
        $this->imageUrl = $data['image_url'] ?? null;
        $this->hint = $data['hint'] ?? null;
        $this->hintOrder = $data['hint_order'] ?? [];
        $this->maxPoints = $data['max_points'] ?? 0;
        $this->mode = $data['mode'] ?? null;
        $this->allowShuffle = $data['allow_shuffle'] ?? false;
        $this->points = $data['points'] ?? null;
    }
}

File: sdk/java/src/main/java/com/hjtpx/captcha/model/ClickCaptchaResponse.java
package com.hjtpx.captcha.model;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

public class ClickCaptchaResponse {
    @JsonProperty("session_id")
    private String sessionId;

    @JsonProperty("image_url")
    private String imageUrl;

    @JsonProperty("hint")
    private String hint;

    @JsonProperty("hint_order")
    private List<Integer> hintOrder;

    @JsonProperty("max_points")
    private int maxPoints;

    @JsonProperty("mode")
    private String mode;

    @JsonProperty("allow_shuffle")
    private boolean allowShuffle;

    @JsonProperty("points")
    private List<List<Integer>> points;

    public String getSessionId() {
        return sessionId;
    }

    public void setSessionId(String sessionId) {
        this.sessionId = sessionId;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }

    public String getHint() {
        return hint;
    }

    public void setHint(String hint) {
        this.hint = hint;
    }

    public List<Integer> getHintOrder() {
        return hintOrder;
    }

    public void setHintOrder(List<Integer> hintOrder) {
        this.hintOrder = hintOrder;
    }

    public int getMaxPoints() {
        return maxPoints;
    }

    public void setMaxPoints(int maxPoints) {
        this.maxPoints = maxPoints;
    }

    public String getMode() {
        return mode;
    }

    public void setMode(String mode) {
        this.mode = mode;
    }

    public boolean isAllowShuffle() {
        return allowShuffle;
    }

    public void setAllowShuffle(boolean allowShuffle) {
        this.allowShuffle = allowShuffle;
    }

    public List<List<Integer>> getPoints() {
        return points;
    }

    public void setPoints(List<List<Integer>> points) {
        this.points = points;
    }
}
